backend/login/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from login.views import login_page, LoginRequest, test
+
+urlpatterns = [
+    url(r'^$', login_page),
+    url(r'^mysnu/', LoginRequest.as_view()),
+    url(r'^test/', test)
```

Comment:
I think test urls better not be implemented if this is just for dev test. Even if purpose of adding this is for future testing, it will be better to use other way to test features instead of providing unnecessary urls.